runelite-api/src/main/java/net/runelite/api/coords/WorldArea.java

```diff
@@ -665,18 +662,15 @@ else if (thisY >= otherY + other.getHeight() - 1)
 		{
 			return false;
 		}
-		return sourceTile.hasLineOfSightTo(targetTile);
+		return sourceTile.hasLineOfSightTo(targetTile) || targetTile.hasLineOfSightTo(sourceTile);
```

Comment:
Does this cause any change of behavior in its current uses? (looks like only the dev tools' scene overlay uses it)

What game behavior does this code change model compared to the original? I would not expect this code to need any change now.